src/controllers/business_page.js

```diff
@@ -1,5 +1,42 @@
+const { getSearchResult } = require('../model/queries/business');
+const { getReviewsByBusiness } = require('../model/queries/review');
 
 exports.get = (req, res) => {
   const { name } = req.query;
-  res.render('business_page', { style: 'style', title: 'business', name });
+  getSearchResult(name)
+    .then((response) => {
+      const {
+        id, address, description, img, category,
+      } = response[0];
+      getReviewsByBusiness(id)
+        .then((responseReview) => {
+          const { avg } = responseReview[0];
+          res.render('business_page', {
+            responseReview,
+            avg,
+            style: 'style',
+            title: 'business',
+            dom: 'business_page',
+            name,
+            address,
+            description,
+            img,
+            category,
+          });
+        })
+        .catch(err => res.render('business_page', {
```

Comment:
Error handling should be done using its own middleware.
Use the ```next``` parameter to handle errors.